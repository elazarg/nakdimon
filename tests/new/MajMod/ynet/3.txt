גִּיל הוּא רַק מִסְפַּר - וְהִנֵּה עוֹד הוֹכָחָה: אידה קילינג זָכְתָה בְּשַׁבָּת הָאַחֲרוֹנָה בְּשִׂיא עוֹלָמִי בְּרִיצָה לִקְבוּצַת הַגִּיל שֶׁלָּהּ. אַגַּב, קילינג הִיא בַּת 101. קילינג סִייְּמָה אֶת ספרינט מֵאָה המטרים בְּתוֹךְ 1:17.33. עִם זֹאת, במירוץ עַצְמוֹ הִיא הִגִּיעָה לְמָקוֹם הַתְּשִׁיעִי בִּלְבַד בֵּין תִּשְׁעַת הרצים. קילינג, תּוֹשֶׁבֶת נְיוּ יוֹרְק, הֵחֵלָּה לָרוּץ בְּגִיל 67. בצעירותה בִּתְקוּפַת הַשֵּׁפֶל בְּאַרְצוֹת הַבְּרִית, הִיא עִסְקָה בְּכָל עֲבוֹדָה אֶפְשָׁרִית, מרחיצת חַלּוֹנוֹת וְעַד שמרטפות, כְּדֵי לְסַיֵּיעַ לכלכלת הַמִּשְׁפָּחָה. בְּגִיל 42 התאלמנה מבעלה, ונותרה מְטוּפֶּלֶת בארבעת יְלָדֶיהָ, כְּשֶׁכּוּלָּם חַיִּים בְּדִירַת חֲדַר אַחַת. בִּשְׁנוֹת הַשִּׁישִּׁים הָפְכָה לפעילה בִּתְנוּעָה לִזְכוּיוֹת אָדָם. "אֲנִי מַרְגִּישָׁה נִפְלָא" אָמְרָה קילינג, "אֲנִי שְׂמֵחָה לְהַצִּיג לְכָל הַקָּהָל הַנֶּהְדָּר שֶׁכָּאן דּוּגְמָא לָמָּה שֶׁאַתָּה יָכוֹל לַעֲשׂוֹת בִּשְׁבִיל עַצְמְךָ, וַאֲנִי מוֹדֶה לֶאֱלוֹהִים כָּל יוֹם עַל שבירך אוֹתִי".