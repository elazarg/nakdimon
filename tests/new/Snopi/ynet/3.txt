גִּיל הוּא רַק מְסַפֵּר - וְהִנֵּה עוֹד הוֹכָחָה: אידה קילינג זָכְתָה בַּשַּׁבָּת הָאַחֲרוֹנָה בַּשִּׂיא עוֹלָמִי בָּרִיצָה לִקְבוּצַת הַגִּיל שֶׁלָּהּ.
אַגַּב, קילינג הִיא בַּת 101.
קילינג סִיְּימָה אֶת ספרינט מֵאָה הַמֶּטְרִים בְּתוֹךְ 1:17.33.
עִם זֹאת, בְּמֵירוֹץ עַצְמוֹ הִיא הִגִּיעָה לַמָּקוֹם הַתְּשִׁיעִי בִּלְבַד בֵּין תִּשְׁעַת הָרָצִים.
קילינג, תּוֹשֶׁבֶת ניו יוֹרֵק, הֵחֵלָּה לָרוּץ בַּגִּיל 67.
בִּצְעִירוּתָהּ בִּתְקוּפַת הַשֵּׁפֶל בְּאַרְצוֹת הַבְּרִית, הִיא עָסְקָה בְּכָל עֲבוֹדָה אֶפְשָׁרִית, מֵרְחִיצַת חַלּוֹנוֹת וְעַד שמרטפות, כְּדֵי לְסַיֵּיעַ לכלכלת הַמִּשְׁפָּחָה.
בַּגִּיל 42 התאלמנה מִבַּעֲלָהּ, וְנוֹתְרָה מְטוּפֶּלֶת בְּאַרְבַּעַת יְלָדֶיהָ, כְּשֶׁכּוּלָּם חַיִּים בְּדִירַת חֶדֶר אַחַת.
בִּשְׁנוֹת הַשִּׁישִּׁים הָפְכָה לִפְעִילָה בַּתְּנוּעָה לִזְכוּיּוֹת אָדָם.
"אֲנִי מֵרְגִישָׁה נִפְלָא" אָמְרָה קילינג, "אֲנִי שְׂמֵחָה לְהַצִּיג לְכָל הַקָּהָל הַנֶּהֱדָר שֶׁכָּאן דּוּגְמָא לָמָּה שֶׁאַתָּה יָכוֹל לַעֲשׂוֹת בִּשְׁבִיל עַצְמְךָ, וַאֲנִי מוֹדֶה לֶאֱלוֹהִים כֹּל יוֹם עַל שֶׁבַּיָּרֵךְ אוֹתִי".