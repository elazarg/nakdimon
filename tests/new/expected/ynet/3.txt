גִּיל הוּא רַק מִסְפָּר - וְהִנֵּה עוֹד הוֹכָחָה: אִידָה קִילִינְג זָכְתָה בַּשַּׁבָּת הָאַחֲרוֹנָה בְּשִׂיא עוֹלָמִי בְּרִיצָה לִקְבוּצַת הַגִּיל שֶׁלָּהּ.
אַגַּב, קִילִינְג הִיא בַּת 101.
קִילִינְג סִייְּמָה אֶת סְפְּרִינְט מֵאָה הַמֶּטְרִים בְּתוֹךְ 1:17.33.
עִם זֹאת, בַּמֵּירוֹץ עַצְמוֹ הִיא הִגִּיעָה לַמָּקוֹם הַתְּשִׁיעִי בִּלְבַד בֵּין תִּשְׁעַת הָרָצִים.
קִילִינְג, תּוֹשֶׁבֶת נְיוּ יוֹרְק, הֵחֵלָּה לָרוּץ בְּגִיל 67.
בִּצְעִירוּתָהּ בִּתְקוּפַת הַשֵּׁפֶל בְּאַרְצוֹת הַבְּרִית, הִיא עָסְקָה בְּכָל עֲבוֹדָה אֶפְשָׁרִית, מֵרְחִיצַת חַלּוֹנוֹת וְעַד שְׁמַרְטַפּוּת, כְּדֵי לְסַיֵּיעַ לְכַלְכָּלַת הַמִּשְׁפָּחָה.
בְּגִיל 42 הִתְאַלְמְנָה מִבַּעֲלָהּ, וְנוֹתְרָה מְטוּפֶּלֶת בְּאַרְבַּעַת יְלָדֶיהָ, כְּשֶׁכּוּלָּם חַיִּים בְּדִירַת חֶדֶר אַחַת.
בִּשְׁנוֹת הַשִּׁישִּׁים הָפְכָה לִפְעִילָה בַּתְּנוּעָה לִזְכוּיוֹת אָדָם.
"אֲנִי מַרְגִּישָׁה נִפְלָא" אָמְרָה קִילִינְג, "אֲנִי שְׂמֵחָה לְהַצִּיג לְכָל הַקָּהָל הַנֶּהֱדָר שֶׁכָּאן דּוּגְמָא לְמָה שֶׁאַתָּה יָכוֹל לַעֲשׂוֹת בִּשְׁבִיל עַצְמְךָ, וַאֲנִי מוֹדֶה לֶאֱלוֹהִים כָּל יוֹם עַל שֶׁבֵּירֵךְ אוֹתִי".
